readme.md

```diff
@@ -119,6 +121,31 @@ Reset the progress back to zero without animating.
 */
 func resetProgress() {}
 
+/**
+Cancels `Progress` if it's set and prevents further updates.
+*/
+func cancelProgress() {}
+
+/**
+Triggers when the progress was cancelled succesfully.
+*/
+var onCancelled: (() -> Void)?
```

Comment:
This needs to be updated if we decide to keep the parameter: https://github.com/sindresorhus/CircularProgress/pull/4#discussion_r250964699

Reply:
Parameter has been removed.